lib/tic_tac_toe_web/controllers/board_controller.ex

```diff
@@ -0,0 +1,19 @@
+defmodule TicTacToeWeb.BoardController do
+    use TicTacToeWeb, :controller
+  
+    def post_move(conn, _params) do
+        conn  
+        |> put_status(:ok)
+        |> send_resp(200, "ok")
+    end
+
+    def fetch_board_update_request(conn) do
```

Comment:
What's `conn` stand for?

Reply:
I will make that more explicit. I thought that was specific to Phoenix Plug, like "send_resp"